src/control/control.cpp

```diff
@@ -150,6 +150,16 @@ QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(
         if (it != s_qCOHash.end()) {
             auto pControl = it.value().lock();
             if (pControl) {
+                auto actualKey = pControl->getKey();
+                if (actualKey != key) {
+                    qWarning()
```

Comment:
I'm unsure about the slowdown this entails, don't we have benchmarks for CO's?

Reply:
I'm not aware of any CO-related benchmarks. Anyway, COs are initialized during startup, so this adds one string comparison per CO at startup and the live performance should be affected. I think this is worth it to be notified of deprecated COs.

Do you have another suggestion? We could check if `--developer` has been passed on the command line.

Reply:
Ah right, yeah its fine then IMO.